Sanya_form.HTML

```diff
@@ -20,47 +20,47 @@
         
         <br/>
         
-      <b><label>Last Name:</label></b>
+      <label>Last Name:</label>
       <input id="last_name" type="text" placeholder="Name" />
         
         <br/>
 
-      <b><label>Date of Birth:</label></b>
+      <label>Date of Birth:</label>
       <input type="date" />
 
         <br/>
         
-      <b><label>Email:</label></b>
+      <label>Email:</label>
       <input type="email" placeholder="email"/>
         
         <br/>
 
-      <b><label>Mobile:</label></b>
-      <input type="mobile" />
+      <label>Mobile:</label>
+      <input type="tel" maxlength="10"/>
         
         <br/>
 
-      <b><label>Address:</label></b>
+      <label>Address:</label>
       <input type="address" maxlength="20" />
 
         <br/>
 
-      <b><label>Gender:</label></b>
+      <label>Gender:</label>
 
         <br/>
 
-      <b><label>Male:</label></b>
-      <input id="male" type="radio" />
+      <label>Male:</label>
+      <input id="male" type="radio" name="gender">
 
         <br/>
 
-      <b><label>Female:</label></b>
-      <input id="female" type="radio" name="gender">
+      <label>Female:</label>
+      <input id="female" type="radio" name="gender" >
 
         <br/>
 
-      <b><label>Other:</label></b>
-      <input id="other" type="radio" name="gender">
+      <label>Other:</label>
+      <input id="other" type="radio" name="gender" >
         
         <br/></b></b>
```

Comment:
Excellent updates, there are just the two closing bold tags that can be removed here and then you are good to go!